src/lib/modes/standard_modes.hpp

```diff
@@ -93,5 +93,36 @@ static inline uint8_t getNavStateFromStandardMode(StandardMode mode)
 	return vehicle_status_s::NAVIGATION_STATE_MAX;
 }
 
+/**
+ * @return returns true for advanced modes
+ */
+static inline bool isAdvanced(uint8_t nav_state)
```

Comment:
This would fit a bit better in this header: https://github.com/PX4/PX4-Autopilot/blob/a0f94bf2a7e06aecf253f64cd709bd26192e40cc/src/lib/modes/ui.hpp

Reply:
done with [53deba8](https://github.com/PX4/PX4-Autopilot/pull/22315/commits/53deba8c845f4b92f4795510a3d43825f4de9566)